redcap/project.py

```diff
@@ -469,7 +469,10 @@ def import_records(self, to_import, overwrite='normal', format='json',
         pl = self.__basepl('record')
         if hasattr(to_import, 'to_csv'):
             # We'll assume it's a df
-            from StringIO import StringIO
```

Comment:
Is there any reason not to do this alongside other top-level imports save my inexperience when first writing this method?